static/scss/dashboard.scss

```diff
@@ -490,6 +490,20 @@
       > p {
         color: $font-black;
       }
+
+      .tos-container {
+        max-height: 200px;
+        overflow-x: hidden;
+        overflow-y: auto;
```

Comment:
also, do you need to set `overflow-x` here? if I disable that style I don't see any visual change.